Documentation/ceph-csi-snapshot.md

```diff
@@ -6,14 +6,17 @@ indent: true
 
 ## Prerequisites
 
-* Requires Kubernetes v1.17+ which supports snapshot beta.
+#### Snapshot API version support matrix
 
-* Install the new snapshot controller and snapshot beta CRD. More info can be found
-[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage)
+| Snapshot API version | Kubernetes Version   | Snapshot-Controller + CRDs Version | Sidecar Version |
+| -------------------- | -------------------- | ---------------------------------- | --------------- |
+| v1                   | k8s >= v1.20         | snapshot-controller >= v4.x        | sidecar >= v2.x |
+| v1beta1              | v1.17 =< k8s < v1.20 | v2.x =< snapshot-controller < v4.x | sidecar >= v2.x |
```

Comment:
For consistency: `<=` instead of `=<`, and I think it will be easier to understand if using `<=` the lesser version instead of `<`
```suggestion
| v1beta1              | v1.17 <= k8s <= v1.19 | v2.x <= snapshot-controller <= v3.x | sidecar >= v2.x |
```

Reply:
have you pushed this change? it still looks the same

Reply:
not yet, I'll push it with all other changes.